test.js

```diff
@@ -135,3 +135,9 @@ test('translate via custom tld', async t => {
     t.is(res.from.text.value, '');
     t.false(res.from.text.didYouMean);
 });
+
+test('translate long text over 2083 chars using any language', async t => {
+    const res = await translate('hi '.repeat(1500), {from: 'english', to: 'dutch'});
+
+    t.truthy(res);
```

Comment:
I think we should check some field inside res to ensure that something is really translated.

Reply:
I'm not sure how that could be implemented, though.

Reply:
I mean to check response fields like in other tests, e.g.
```js
t. truthy(res.text.indexOf('Hoi') >= 0);
...

Reply:
I'll try putting that particular field in to start with and see if that helps much.